fuel/app/classes/materia/widget/instance/manager.php

```diff
@@ -64,6 +64,22 @@ public static function get_all_for_user($user_id)
 		else return [];
 	}
 
+	/**
+	 * Checks to see if the given widget instance is locked for the current user.
+	 *
+	 * @param inst_id widget instance id to check for lock
+	 *
+	 * @return bool whether the given instance is locked for the current user
+	 */
+	public static function locked_by_current_user(string $inst_id): bool
```

Comment:
It can technically be used either way, since 'true' means the widget is locked BY the current user and 'false' means the widget is locked FOR the current user, but you're right that the doc code should match the vocabulary.

Reply:
name seems ok to me now.  current docs mean what it does to me.